src/Proj4.jl

```diff
@@ -50,14 +48,21 @@ function _get_def(proj::Projection)
 end
 
 """Low level interface to libproj transform, allowing user to specify strides"""
-function _transform!(src::Projection, dest::Projection, point_count, point_stride, x, y, z)
+function _transform!(src::Projection, dest::Projection, point_count::Int64,
+                     point_stride::Int32, x::Ptr{Float64}, y::Ptr{Float64}, z::Ptr{Float64})
     @assert src.rep != C_NULL && dest.rep != C_NULL
     ccall((:pj_transform, libproj), Cint,
           (Ptr{Void}, Ptr{Void}, Clong, Cint, Ptr{Cdouble}, Ptr{Cdouble}, Ptr{Cdouble}),
           src.rep, dest.rep, point_count, point_stride, x, y, z)
 end
 
-
+function _transform!(src::Projection, dest::Projection, point_count::Int64,
+                     point_stride::Int32, x::Ptr{Float64}, y::Ptr{Float64})
+    @assert src.rep != C_NULL && dest.rep != C_NULL
+    ccall((:pj_transform, libproj), Cint,
+          (Ptr{Void}, Ptr{Void}, Clong, Cint, Ptr{Cdouble}, Ptr{Cdouble}, Ptr{Void}),
+          src.rep, dest.rep, point_count, point_stride, x, y, C_NULL)
+end
```

Comment:
This is for dealing with calls to transforms for 2-dimensional points, without having to pass in `C_NULL` for z (otherwise it will be unclear whether it is a `Ptr{Float64}` or a `Ptr{Void}`)